### PR TITLE
fmt: go fmt project

### PR DESCRIPTION
**Pull request by dpastoor**
go fmt project and cleaned up some misc top level files that were legacy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,13 +18,13 @@ command line interface for executing and managing models and projects. The nomen
 
 For Example:
 
-`bbi nonmem run sge /abs/path/to/file.mod`
+`bbi nonmem run sge path/to/file.mod`
 
  * `nonmem` : The modeling software we should be targeting for this run
  * `sge` : The mode of execution. For nonmem this can either be local or sge, with sge indicating submission of jobs to the grid
  * `/abs/path/to/file.mod` : The location of the file to submit for execution. Can be relative or absolute.
```
Comment (seth127) [inline README.md:25]: This should probably be `path/to/file.mod` too for consistency

File: README.md [hunk 2]
```diff
@@ -58,122 +52,20 @@ For Example:
...
-                        "V2",                                         
-                        "RUVp",                                       
-                        "RUVa"                                        
-                ],                                                    
-                "Omega": [],                                          
-                "Sigma": []                                           
-        },                                                            
-        "OFV": 2643.561                                               
-}    
+
```
Comment (seth127) [inline README.md:62]: Both of these examples appear to be blank... Did you want to replace those with something or should we scrap this whole section?

File: README.md [hunk 3]
```diff
@@ -58,122 +52,20 @@ For Example:
...
-|   Theta   |   Name   |   Estimate (SN)   |   Estimate   |      StdErr (RSE)   |
-+-----------+----------+-------------------+--------------+---------------------+
-|   TH 1    |     KA   |        2.31E+00   |       2.31   |    0.0862 (3.7 %)   |
-|   TH 2    |     CL   |        5.50E+01   |         55   |      3.33 (6.1 %)   |
-|   TH 3    |     V2   |        4.65E+02   |        465   |      29.7 (6.4 %)   |
-|   TH 4    |   RUVp   |       -8.06E-02   |    -0.0806   |   0.0555 (68.9 %)   |
-|   TH 5    |   RUVa   |        4.13E+00   |       4.13   |     1.36 (32.9 %)   |
-+-----------+----------+-------------------+--------------+---------------------+
+
 ```
```
Comment (seth127) [inline README.md:71]: Is the section below still correct? I don't think so, but I could be wrong. If it's not, let's just remove it for now.

**Review by seth127 — CHANGES_REQUESTED** [review]
I like this housekeeping PR and I'd like to merge it, but I'm not sure if you were done with it or not. Either way, there are one or two things that appear unfinished.

**Comment by seth127** [comment]
related: is this branch stale? It looks like most, if not all, of the changes from it have been done in this branch now
https://github.com/metrumresearchgroup/bbi/pull/200

If so, let's close and delete the other one from December.

**Comment by dpastoor** [comment]
Thanks for the feedback!

**Comment by dpastoor** [comment]
> related: is this #200 branch/PR stale? It looks like most, if not all, of the changes from it have been done in this branch now.
> 
> If so, let's close and delete the other one from December.

Yes I followed the same procedure here as that other branch did not want to attempt to reconcile with the naming changes from the previous one so figured would just start a new one Will close #200 out

**Review by seth127 — APPROVED** [review]
Looks good

**Pull request merged**